src/common/entity/can_toggle_state.ts

```diff
@@ -6,9 +6,17 @@ import { supportsFeature } from "./supports-feature";
 
 export const canToggleState = (hass: HomeAssistant, stateObj: HassEntity) => {
   const domain = computeStateDomain(stateObj);
+
   if (domain === "group") {
-    return stateObj.state === "on" || stateObj.state === "off";
+    for (const entity of stateObj.attributes.entity_id) {
+      const d = computeStateDomain(hass.states[entity]);
+      if (canToggleDomain(hass, d)) {
+        return stateObj.state === "on" || stateObj.state === "off";
```

Comment:
should this be the state of `entity`?

Reply:
No its should be the main group
This basically checks if toggle is possible and then it should use the group state for the on and off 
otherwise it would be different that the previous functionality when toggle is present